03_gamedesign/index.tex

```diff
@@ -1,5 +1,75 @@
+        \item Check if a disaster occurs this turn.
+        \item Server processes actions and updates game and island states.
+            \begin{itemize}
+                \item A cost of living is subtracted from an islands pool before the next term. This is the simulation-level equivalent to using resources to stay alive (e.g. food consumed). These resources are permanently consumed and do NOT go into the common pool. Note: this is NOT the same as the tax.
+                \item Check if the game is over.
+                \item Check if any islands are \textbf{critical} (i.e. below the threshold).
+                \item Check if any islands are \textbf{dead}.
+            \end{itemize}
+    \end{enumerate}        
+\end{definition}
+
+\begin{definition} \label{def:gameseason}
+    A \textbf{season} is defined as a series of turns and concludes with a disaster. Seasons formalise the flow of the game and provide a method to track the number of disasters the islands survive.
+\end{definition}
+
+\begin{definition} \label{def:gamestate}
+    The \textbf{game state} is the set of information an island receives at the start of each turn. This includes, but is not limited to, the resources it was allocated the previous turn, the geographical location of the islands, the quantity of resources left in the common pool and the set of rules for the turn (including any rules that were modified the previous turn). It is important to note that each island's game state is private, so no other island can see another island's resource allocation.
+\end{definition}
+
+\begin{definition} \label{def:gameaction}
+    An \textbf{action} is a decision an island can make or that is made by any Inter-Island Organisation that updates the game state. Actions can be made at different stages within a turn both within Inter-Island Organisations and at the end of a turn. This includes but is not limited to taking resources from or donating resources to from the common pool, rule changes in the IIGO, and gift requests or acceptances. 
+\end{definition}
+
+\begin{definition} \label{def:critical}
+    A \textbf{critical} island is defined as an island whose resources are below the minimum threshold. When this occurs, an island is allowed a grace period during which it is expected to request gifts from other islands in order to reach the minimum amount of resources required to stay in the game.
+\end{definition}
+
+\begin{definition} \label{def:death}
+    A \textbf{death} occurs when an island was in a \textbf{critical} state for $N$ turns. The exact number of turns affects the difficulty of the game so it was investigated and is discussed in simulations and results section. Once an island is dead, it can no longer participate in the simulation.
+\end{definition}
+
+\begin{definition} \label{def:gameover}
+    \textbf{Game over} is defined as the end of the simulation and occurs when all islands have died or the simulation is completed - whichever occurs first.

```

Comment:
```suggestion
    \textbf{Game over} is defined as either the end of the simulation and occurs when all islands have died or the simulation is completed, whichever occurs first.

```

Reply:
the dash here is more appropriate grammatically so will keep :)